exonum-java-binding-core/src/main/java/com/exonum/binding/transaction/RawTransaction.java

```diff
@@ -0,0 +1,55 @@
+package com.exonum.binding.transaction;
+
+import com.google.auto.value.AutoValue;
+
+/**
+ * An Exonum raw transaction. It is mainly used for interaction with the Exonum core
```

Comment:
I don't like that this Javadoc is too abstract (half the classes are used for interaction with Exonum core).

Reply:
I'd suggest this:

```
/**
 * An Exonum raw transaction. The raw transaction is different from {@link TransactionMessage}
 * as it only includes the serialized transaction parameters and transaction identifiers.
 * Author’s public key is <em>not</em> included but is accessible from
 * the {@linkplain TransactionContext#getAuthorPk() execution context}.
 *
 * <p>A raw transaction is converted to an {@linkplain Transaction executable transaction}
 * by the framework using an implementation of {@link TransactionConverter}.
 */
```

Reply:
Also, in Javadocs we use a lone `<p>`, not `</br>`